posts/2023-12-12-23.0.0.12.adoc

```diff
@@ -0,0 +1,533 @@
+alpha_timer_seconds{scope="application",quantile="0.8",} 5.603590144
+alpha_timer_seconds_bucket{scope="application",le="0.1",} 0.0 <2>
+alpha_timer_seconds_bucket{scope="application",le="0.2",} 0.0 <2>
+alpha_timer_seconds_bucket{scope="application",le="1.0",} 1.0 <2>
+alpha_timer_seconds_bucket{scope="application",le="+Inf",} 2.0  <2><3>
+alpha_timer_seconds_count{scope="application",} 2.0
+alpha_timer_seconds_sum{scope="application",} 6.333
+----
+
+<1> The Prometheus metric type is `histogram`. Both the quantiles or percentiles and buckets are represented under this type.
+<2> The `le` tag represents _less than_ and is for the defined buckets, which are converted to seconds.
+<3> Prometheus requires a `+Inf` bucket, which counts all hits.
+
+Additionally, the `@RegistryScope` annotation is now a CDI qualifier.
+
+For more information about MicroProfile Metrics, see:
+
+* link:https://github.com/eclipse/microprofile-metrics[MicroProfile Metrics repo]
+* link:{url-prefix}/docs/latest/introduction-monitoring-metrics.html[Monitoring with metrics documentation]
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC>
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // //
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/26945
+// Contact/Reviewer: yasmin-aumeeruddy
+// // // // // // // //
+[#mpt11]
+=== Expanded Java EE and Jakarta EE support in MicroProfile Telemetry 1.1
+
+MicroProfile Telemetry 1.1 provides you with the latest Open Telemetry technology as the feature now consumes OpenTelemetry-1.29.0, which is updated from 1.19.0.
+
+The feature is compatible with the following programming model combinations:
+
+* Java EE 7 paired with MicroProfile 1.4
+* Java EE 8 paired with MicroProfile 4.1
+* Jakarta EE 9 paired with MicroProfile 5.0
+* Jakarta EE 10 paired with MicroProfile 6.1.
+
+To enable this feature, add the following feature definition to your `server.xml` file:
+
+[source,xml]
+----
+<features>
+   <feature>mpTelemetry-1.1</feature>
+</features>
+----
+
+Also, you must make third-party APIs visible for your application in the `server.xml` file:
+
+[source,xml]
+----
+<webApplication location="demo-microprofile-telemetry-inventory.war" contextRoot="/">
+    <!-- enable visibility to third party apis -->
+    <classloader apiTypeVisibility="+third-party"/>
+</webApplication>
+----
+
+[#trace]
+=== Trace incoming HTTP requests with MicroProfile Telemetry 1.0 and 1.1
+Also in Open Liberty 23.0.0.12, the MicroProfile Telemetry 1.0 and 1.1 features are enhanced to automatically trace incoming HTTP requests (static files, servlets, and JSPs).
+
+For more information about MicroProfile Telemetry, see the following links:
+
+* link:https://github.com/open-telemetry/opentelemetry-specification/blob/v1.20.0/specification/trace/api.md[Tracing API]
+* link:https://openliberty.io/docs/latest/docs/latest/microprofile-telemetry.html[Enable distributed tracing with MicroProfile Telemetry]
```

Comment:
These links aren't formatted properly in the draft site

Reply:
same here- fixed in [apply suggestions from review](https://github.com/OpenLiberty/blogs/pull/3470/commits/c14691fdc2a11d3aa2a1aa7e7bbc08d9e0d76dc6)